rotkehlchen/chain/polygon_pos/tokens.py

```diff
@@ -0,0 +1,20 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.chain.evm.tokens import EvmTokens
+from rotkehlchen.constants.assets import A_POLYGON_POS_MATIC
+from rotkehlchen.types import ChecksumEvmAddress
+
+if TYPE_CHECKING:
+    from rotkehlchen.db.dbhandler import DBHandler
+
+    from .node_inquirer import PolygonPOSInquirer
+
+
+class PolygonPOSTokens(EvmTokens):
+
+    def __init__(self, database: 'DBHandler', polygon_pos_inquirer: 'PolygonPOSInquirer') -> None:
+        super().__init__(database=database, evm_inquirer=polygon_pos_inquirer)
+
+    # -- methods that need to be implemented per chain
+    def _per_chain_token_exceptions(self) -> list[ChecksumEvmAddress]:
+        return [A_POLYGON_POS_MATIC.resolve_to_evm_token().evm_address]
```

Comment:
Yes. If you query `balanceOf` of this token https://polygonscan.com/address/0x0000000000000000000000000000000000001010#readContract#F6 for some address, it will return the amount of MATIC owned by that address on polygon.